life.js

```diff
@@ -1,9 +1,10 @@
 const lifeBeyond = async () => {
   const character = await fetchCharacter();
-  const lifeDomain = character.classes.filter((characterClass) => characterClass.subclassDefinition.name == 'Life Domain')[0];
+  const lifeDomain = character.classes.filter((charClass) => charClass.subclassDefinition.name == 'Life Domain')[0];
```

Comment:
In this case, `characterClass` is a variable name specified by us, rather than something that was specified by dndbeyond in their DOM. 

For more information on this syntax, please see the [ES6 arrow function page on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) which I think gives a nice explanation.